### PR TITLE
Add web_url

### PR DESCRIPTION
**Pull request by vladsavelyev**
Add function to get CPG web URL. URL template is parametrised by an environment variable, similar to other CPG paramers. The variable is added into the server AR code: https://github.com/populationgenomics/analysis-runner/pull/411

~Make sure to merge https://github.com/populationgenomics/cpg-utils/pull/36 first~

### CODE REVIEW

**Review by vivbak — APPROVED** [review]
🙏  Thanks!!

File: cpg_utils/hail_batch.py [hunk 1]
```diff
@@ -17,6 +17,8 @@
     'gcloud -q auth activate-service-account --key-file=/gsa-key/key.json'
 )
 
+WEB_URL_FORMAT = 'https://{namespace}-web.populationgenomics.org.au/{dataset}'
```
Comment (lgruen) [inline cpg_utils/hail_batch.py]: Should we make this configurable or will `web_url` only be called for CPG-internal implementations?

**Review by lgruen — CHANGES_REQUESTED** [review]

File: cpg_utils/hail_batch.py (hunk 1 above)
Comment (vladsavelyev) [reply cpg_utils/hail_batch.py]: Currently in the cpg-pipes, `CpgStorageProvider` uses the `web_url` function from cpg_utils, and the abstract one doesn't have a default implementation.

But if we are getting rid of those abstractions and relying entirely on the toml config to configure the environment, then we should somehow explain that the WEB URL value should be parametrised by namespace and dataset. It's straightforward to put such requirements with abstract classes, but a bit more error-prone with a string template like that. But could add `web_url_template` into the toml config with a comment.

Comment (vladsavelyev) [reply cpg_utils/hail_batch.py]: Added a commit to parametrise it with an environment variable

**Review by lgruen — APPROVED** [review]

**Pull request merged**